.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -403,7 +403,7 @@ stages:
         build: true
         target: builder
         baseImage: debian
-        command: "Clean BuildTracerHome"
+        command: "Clean BuildTracerHome ZipMonitoringHome"
```

Comment:
🤔 I don't think this should be here. Rebasing issue?

Actually, I see the `build_tracer_home` has it too - I think they should _both_ not have it, as this is done in `package_arm64`?

Reply:
yeah that's a rebase issue. Thx will fix it